witness/witness/cmd/witness/example_config.yaml

```diff
@@ -0,0 +1,4 @@
+Logs: 
+    Description: Google 'Argon2021' log
+    LogID: 9lyUL9F3MCIUVBgIMJRWjuNNExkzv98MLyALzE7xZOM=
```

Comment:
If the `LogID` is going to be set to `log.ID` (default value) then it's best to just not put it in the config. It's only asking for a mistake.